ansible/roles/test/vars/testcases.yml

```diff
@@ -127,6 +127,12 @@ testcases:
       filename: link_flap.yml
       topologies: [t0, t0-16, t0-64, t0-64-32, t0-116, t1, t1-lag, t1-64-lag, ptf32, ptf64]
 
+    continuous_link_flap:
+      filename: continuous_link_flap.yml
+      topologies: [t1, t1-lag, t1-64-lag]
+      required_vars:
```

Comment:
>required_vars [](start = 6, length = 13)

Is it required or not? I see the default value.

Reply:
I added in this file, because I want to point this out, that it is required var. 
Because even though I assign it by default to 10%,  as you pointed out earlier, on some platform orchagent CPU never comes below 10, so test may fail.

Reply:
I see your point, but It is ok to be optional.

---
In reply to: [278343854](https://github.com/Azure/sonic-mgmt/pull/810#discussion_r278343854) [](ancestors = 278343854)